content/tutorial-deep-reinforcement-learning-with-pong-from-pixels.ipynb

```diff
@@ -0,0 +1,1275 @@
+{
```

Comment:
<p>Same note about installation goes here. I think because you mentioned matplotlib in the prerequisites section, showing again how to install it might be unnecessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Added Matplotlib to prereq and removed the `pip` step from that section. Thank you!